parsl/executors/high_throughput/executor.py

```diff
@@ -188,6 +189,7 @@ def __init__(self,
         self.mem_per_worker = mem_per_worker
         self.max_workers = max_workers
         self.prefetch_capacity = prefetch_capacity
+        self.all_addresses = ','.join(get_all_addresses())
```

Comment:
respect the address parameter still - almost definitely som people are going to need to override this behavior.

if address is specified, don't do any get_all_addresses magic at all

Reply:
This is a bit that I wanted to ask folks for comments. I'm leaning towards `address=None` will be the default and that implies we try auto-detecting vs, `address=X` means we will use only the user's recommendation.

Reply:
i think that is fine.

Reply:
Fixed.